core/src/main/java/org/apache/struts2/util/StrutsUtil.java

```diff
@@ -50,32 +61,30 @@ public class StrutsUtil {
 
     protected HttpServletRequest request;
     protected HttpServletResponse response;
-    protected Map<String, Class> classes = new Hashtable<>();
-    protected OgnlTool ognl;
+    protected Map<String, Class<?>> classes = new HashMap<>();
```

Comment:
This class is constructed per request thread so I don't believe we need concurrency handling here